src/lib/core/fetch/resolveCoreAccount.ts

```
@@ -0,0 +1,63 @@
+import {
```

ESLint

[error] Filename is not in kebab case. Rename it to `resolve-core-account.ts`.

(unicorn/filename-case)